source/FreeRTOS_DNS.c

```diff
@@ -180,8 +180,10 @@
     #include "pack_struct_end.h"
     typedef struct xDNSTail DNSTail_t;
 
+    #if ( ipconfigUSE_IPv4 != 0 )
```

Comment:
Please run uncrustify, a lot of palces needs it.

Reply:
Uncrustify is run and the formatting issues are fixed already for this PR. These indentations were kept the way it is to reduce unnecessary diff in for the indentation changes as most of these flags are covering lots of lines, also I'm not sure if we should indent those many lines, these could shift a lot of lines so much spaces to the right, making readability hard.

Reply:
I agree, i was commenting with the understanding that uncrustify check might also be failing. So uncrustify do not indent for #if ?